static/app/utils/useTeams.tsx

```diff
@@ -125,6 +134,69 @@ function useTeams({limit}: Options = {}) {
     fetchError: null,
   });
 
+  const slugsRef = useRef<Set<string>>(new Set(slugs));
```

Comment:
Does it actually need the type if you're already passing it

Reply:
o woops it shouldn't